js/components/skjema/TekstfeltMedEnhet.js

```diff
@@ -4,13 +4,14 @@ import cn from 'classnames';
 import { fieldPropTypes } from '../../propTypes';
 import Feilmelding from './Feilmelding';
 
-const TekstfeltMedEnhet = ({ label, id, input, meta }) => {
+const TekstfeltMedEnhet = ({ label, id, input, meta, undertekst }) => {
+    const inputType = undertekst === 'prosent' ? 'tel' : 'text';
```

Comment:
Er `tel` et triks for at bruker kun kan oppgi heltall?

Reply:
@knutnyg Det er et triks som gir numerisk tastatur på mobil :)

Reply:
Det var det ja 🙆‍♂️